### PR TITLE
Add ADFS support

### PR DESCRIPTION
**Pull request by Frycos**
We added an ADFS plugin for this helpful tool. This was successfully tested against a bunch of live systems of course, works like a charm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ All other usage details can be found [here](https://github.com/knavesec/CredMast
 - Oliver Morton ([grimhacker](https://twitter.com/grimhacker)) - Office365UserEnum tool
 - Marcello ([byt3bl33d3r](https://twitter.com/byt3bl33d3r)) - SprayingToolkit
 - Erforschr - HTTP Bruteforce tool
+- Florian Hauser ([frycos](https://twitter.com/frycos) from [codewhitesec](https://twitter.com/codewhitesec)) - ADFS plugin
```
Comment (knavesec) [inline README.md:57]: Looks good, add the ADFS plugin to the list above in the readme/wiki

File: plugins/adfs/adfs.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        'mkt' : '',
+        'lc' : ''
+    }
+
+    spoofed_ip = generate_ip()  # maybe use client related IP address
+    amazon_id = generate_id()
+    trace_id = generate_trace_id()
+
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:65.0) Gecko/20100101 Firefox/65.0', #useragent,
```
Comment (knavesec) [inline plugins/adfs/adfs.py:53]: user agent should be the input, not static

**Review by knavesec — CHANGES_REQUESTED** [review]
Two comments to be addressed, did some limited testing, overall looks good though!

File: plugins/adfs/adfs.py (hunk 2 above)
Comment (Frycos) [reply plugins/adfs/adfs.py]: Absolutely. Provided the change a few minutes ago. Note: the CredMaster's default user agent did not work for my ADFS targets. This is why I changed it but I will give this hint in the wiki documentation then instead.

File: README.md (hunk 1 above)
Comment (Frycos) [reply README.md:58]: Added entry to README list. We've to find a PR solution for the wiki content.

**Pull request merged**